l1-contracts/src/core/Rollup.sol

```diff
@@ -856,14 +934,12 @@ contract Rollup is EIP712("Aztec Rollup", "1"), Leonidas, IRollup, ITestRollup {
     SignatureLib.Signature[] memory _signatures,
     bytes32 _digest,
     Timestamp _currentTime,
-    bytes32 _txEffectsHash,
+    bytes32 _blobHash,
```

Comment:
Same down here, there is a bunch of `blobHash` that should probably be `blobsHash`.